drivers.json

```diff
@@ -43,7 +43,11 @@
 		]
 	},
 	"i2c": {
-		"compatible": ["i2c-gpio"]
+		"compatible": ["efinix,efx-i2c"],
+		"private_data": [
+			"clock-frequency = <100000>;",
```

Comment:
Is the user required to manually set the clock-frequency if the system clock is not 100MHz?

Reply:
Yes. 100KHz is the default value for I2C clock. So, if user want to use 400KHz or 1MHz they need to modify it in linux.dts file.

Reply:
I see, so this data is to set the I2C output clock. If that so, then we can close this.